engine/src/main/java/com/sandboni/core/engine/Arguments.java

```diff
@@ -27,4 +27,5 @@
     private String alwaysRunAnnotation;
     private String seloniFilePath;
     private boolean enablePreview;
+    private boolean runSelectiveModeOnReflectionCall;
```

Comment:
for me it make more sense to add reflection decision into the risk level that we are planning

Reply:
It makes sense, but we need to add it to the proposal.
I can remove this property when we implement the risk level

Reply:
removed